configdialog.cpp

```diff
@@ -1530,3 +1547,128 @@ LRESULT CALLBACK AddPredefinedPaletteDialog(HWND dialog, UINT message, WPARAM wp
 
 	return FALSE;
 }
+
+LRESULT CALLBACK ScreenSaverPatternSelectorDialog(HWND dialog, UINT message, WPARAM wparam, LPARAM lparam) {
+	static std::vector<std::pair<PatternName, std::wstring>> disabled_patterns = { };
```

Comment:
I think I started this trend, but looking at it now storing important state as statics in a window procedure function is starting to feel kind of icky. If we ever have two windows that use these proc functions they will each be reading from the same data and that might be wrong. Maybe. Something to think about later.